src/main/java/org/usfirst/frc/team4373/robot/input/hid/RooJoystick.java

```diff
@@ -97,8 +91,8 @@ public double getAxis(int axis) {
      * @return the joystick angle
      */
     public double getAngle() {
-        double x = this.getAxis(AxisType.kX);
-        double y = this.getAxis(AxisType.kY);
+        double x = this.getAxis(0); //kX
```

Comment:
Instead of this, would recommend implementing our own enum type that maps to the int values used in `getAxis`. Random floating constants are hard to maintain.

Reply:
@aaplmath That seems like a good idea